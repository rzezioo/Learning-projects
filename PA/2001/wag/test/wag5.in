20
275 10
113 7
1 1
3 3
413 9
1 1
28 5
3 3
1 3
44 10
937 10
299 9
3 2
127 9
1 1
1 3
36 6
1 1
206 8
20 6
