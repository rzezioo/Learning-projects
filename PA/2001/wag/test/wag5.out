44 10
1 3
1 3
127 9
275 10
20 6
3 3
3 3
1 1
1 1
1 1
1 1
36 6
299 9
3 2
206 8
413 9
28 5
113 7
937 10
